sp800-63b/sec1_2_introduction.md

```diff
@@ -4,29 +4,29 @@
 
 _This section is informative._
 
-This document and its companion documents, [[Special Publication (SP) 800-63-3]](sp800-63-3.html), [[SP 800-63A]](sp800-63a.html), and [[SP 800-63C]](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
+This document and its companion documents, [Special Publication (SP) 800-63](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
```

Comment:
Agreed. Wasn't sure how we wanted to do that but I think it's a good idea. There's a related issue with making references directly to section numbers. We can make sure the reftags persist in a revision, but where we have section numbers in the text that point across volumes we could run in to problems down the line.